deploy/charts/crate-operator-crds/templates/cratedbs-cloud-crate-io.yaml

```diff
@@ -291,12 +291,44 @@ spec:
                       snapshot:
                         description: The name of the snapshot to use.
                         type: string
+                      type:
+                        type: string
+                        enum: ["all", "tables", "metadata", "partitions", "sections"]
```

Comment:
It's not entirely clear to me from the crate docs how "sections" works. Can you restore sections on their own, or do we have to combine this with restoring metadata, and then specifying which section of metadata to restore (i.e. `udfs`)?

Reply:
As far as I understand you can restore one (or more) data sections on their own. e.g. `RESTORE SNAPSHOT repository_name.snapshot_name TABLES,VIEWS,USERS,PRIVILEGES,...`. It also says `A section cannot be combined with the ALL, METADATA, or TABLE keywords.` in the docs.